_targets.R

```diff
@@ -59,6 +59,9 @@ mainstem_reaches_tidal <- c("2771_1","2769_1","2768_1","2767_1","2764_1","2762_1
 # Define the url for the NHGFv1 to NHDv2 crosswalk
 GFv1_NHDv2_xwalk_url <- "https://raw.githubusercontent.com/USGS-R/drb-network-prep/main/2_process/out/GFv1_NHDv2_xwalk.csv"
 
+# Define the url for the edited HRU polygons
+GFv1_HRUs_edited_url <- "https://github.com/lekoenig/drb-network-prep/blob/c45f469098dfb245a2d686b807a58afa00b9d0b2/2_process/out/GFv1_catchments_edited.gpkg?raw=true"
```

Comment:
I think we should change this to USGS-R main instead of your personal repo.
```suggestion
GFv1_HRUs_edited_url <- "https://github.com/USGS-R/drb-network-prep/blob/c45f469098dfb245a2d686b807a58afa00b9d0b2/2_process/out/GFv1_catchments_edited.gpkg?raw=true"
```

I'm thinking it's better to use a specific commit (`/blob/c45f469098dfb245a2d686b807a58afa00b9d0b2/`) than to specify main (`/blob/main/`), which would always grab the latest commit. Specifying a specific commit will always download the same file. And if we want to grab a later version, changing the link to that commit will cause the target to rebuild due to the link change.